challenges-webapp/src/main/webapp/vue-app/engagement-center/components/programs/Programs.vue

```diff
@@ -0,0 +1,58 @@
+<!--
+This file is part of the Meeds project (https://meeds.io/).
+Copyright (C) 2022 Meeds Association
+contact@meeds.io
+This program is free software; you can redistribute it and/or
+modify it under the terms of the GNU Lesser General Public
+License as published by the Free Software Foundation; either
+version 3 of the License, or (at your option) any later version.
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+Lesser General Public License for more details.
+You should have received a copy of the GNU Lesser General Public License
+along with this program; if not, write to the Free Software Foundation,
+Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+-->
+<template>
+  <div
+    id="EngagementCenterPrograms"
+    class="border-box-sizing"
+    role="main"
+    flat>
+    <v-toolbar
+      color="transparent"
+      flat
+      class="pa-4">
+      <div class="border-box-sizing clickable addChallengeButton">
+        <v-btn class="btn btn-primary">
+          <v-icon>fas fa-plus</v-icon>
+          <span class="mx-2 d-none d-lg-inline">
+            {{ $t('programs.button.addProgram') }}
+          </span>
+        </v-btn>
+      </div>
+      <v-spacer />
+    </v-toolbar>
+    <programs-list class="my-10 mx-4" :programs="programs" />
+  </div>
+</template>
+
+<script>
+export default {
+  data() {
+    return {
+      programs: null
+    };
+  },
+  created() {
+    this.getAllPrograms();
```

Comment:
```suggestion
    this.retrievePrograms();
```

Using `get` in method name means that the method will return the desired object without implicit alteraion of current component attributes (this is confusing with Getters & Setters methods definition). Thus you need to name this method differently, `init`, `retrieveXYZ` ... but not using `get`